src/etc/inc/captiveportal.inc

```diff
@@ -347,15 +347,10 @@ EOD;
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-error.html");
 		@unlink("{$g['varetc_path']}/captiveportal-{$cpzone}-logout.html");
 
-		captiveportal_radius_stop_all();
+		captiveportal_disconnect_all(10); // NAS-Request
 
 		captiveportal_filterdns_configure();
 
-		/* send Accounting-Off to server */
-		if (!platform_booting()) {
-			captiveportal_send_server_accounting(true);
```

Comment:
Why remove this part? It seems to be unrelated to the other changes, unless I'm overlooking something.